src/chronos/utils/date-compare.ts

```diff
@@ -33,6 +33,20 @@ export function isBefore(
   return endOf(date1, units).valueOf() < date2.valueOf();
 }
 
+export function isDisabledDay(date: Date, daysDisabled: number[]): boolean {
+  if (daysDisabled === undefined || !daysDisabled.length) {
+    return false;
+  }
+
+  return (
+    daysDisabled.some(
```

Comment:
```suggestion
    daysDisabled.some((day: number) => day === date.getDay());
```
and remove what's benith
wrote by @valorkin

Reply:
@ludmilanesvitiy 
Dima was not talking about removing, but about replacing filter on some method.